src/main/java/mate.academy/Main.java

```diff
@@ -10,7 +10,8 @@ public class Main {
     public static void main(String[] args) {
         // Please test your Injector here. Feel free to push this class as a part of your solution
         Injector injector = Injector.getInjector();
-        ProductService productService = null;
+        ProductService productService = (ProductService) injector.getInstance(
+                ProductService.class);
```

Comment:
The casting of `injector.getInstance(ProductService.class)` to `ProductService` can lead to a `ClassCastException`. Ensure that the `Injector` is configured to return the correct implementation for `ProductService.class` to avoid runtime errors.